### ISSUE TITLE
Onboarding - customizing appearance - homepage template: "go shopping" button doesn't work

### ISSUE THREAD

**Issue by pmcpinto** [woocommerce/woocommerce-admin#3625]
**Describe the bug**
Nothing happens when I click on the "Go shopping" button that we display on the homepage that's created during the onboarding. I'm having the same problem with the 2 homepage templates: single product and multiple products.

**To Reproduce**
1. Create a store with the new onboarding
2. Visit the Customize Appearance task in the task list
3. Click to proceed with the install of sample products
4. Click to create a homepage
5. Visit the homepage page editor
6. Preview the homepage and click in the "go shopping button"

**Expected behavior**
It should redirect the user to the shop page

**Screenshots**
https://d.pr/i/3etuFu

**Comment by joshuatf** [comment]
@pmcpinto I think this might be left open-ended for the user to set up.  Do you think we should link it to the `Shop` page by default?

**Comment by pmcpinto** [comment]
> @pmcpinto I think this might be left open-ended for the user to set up. Do you think we should link it to the Shop page by default?

I think it would be better to default the link to the shop page and then the user can change it, instead of leaving it blank.

I tested the onboarding flow ~10 times or more and just in one the last tests I noticed that the link was empty. Considering that all the other buttons on the homepage have a link to products or categories by default, I was expecting to have the same for the "go shopping" button.

I'm assuming that merchants are going to have more attention to this type of detail when they set up their store, but I feel that it can be easy to miss this detail. Makes sense?
